src/robot/output/listeners.py

```diff
@@ -74,6 +74,10 @@ def _import_listeners(self, listener_data):
                              % (listener, unicode(err)))
         return listeners
 
+    def imported(self, name, import_type):
+        for listener in self._listeners:
+            listener.call_method(listener.imported, name, import_type)
```

Comment:
1. Should have `import_type` argument first and `name` second to be consistent with `output_file` method (where `name` argument should really be called `file_type`).

2. Should call method with third argument `attrs` that could be an empty dictionary at this point. It would allow us adding more information to the call by adding items to the `attrs` dictionary similarly as we've done with `start_xxx` and `end_xxx` method. Such information could include arguments to the imported thing, source where it was imported from, etc.